core/signers/signers.go

```diff
@@ -75,27 +76,27 @@ func Path(s *Signer, ks keySpace, itemIndexes ...uint64) [][]byte {
 }
 
 // Create creates and stores a Signer in the database
-func Create(ctx context.Context, db pg.DB, typ string, xpubs []string, quorum int, clientToken string) (*Signer, error) {
+func Create(ctx context.Context, db pg.DB, typ string, xpubs []chainkd.XPub, quorum int, clientToken string) (*Signer, error) {
 	if len(xpubs) == 0 {
 		return nil, errors.Wrap(ErrNoXPubs)
 	}
 
-	sort.Strings(xpubs) // this transforms the input slice
+	sort.Sort(sortKeys(xpubs)) // this transforms the input slice
```

Comment:
unless you can think of examples otherwise, I think since numbers are "less than" letters, it should be functionally equivalent. By byte is probably the more accurate sort that we want anyhow.